.github/workflows/build_docker.yaml

```diff
@@ -0,0 +1,40 @@
+name: Docker
```

Comment:
Figure this Action can have other jobs added on for other containers, like `prime-router_builder`.

Reply:
That is a brilliant idea, YES!